pages/editForm/index.js

```diff
@@ -112,6 +127,20 @@ export default function EditForm({ item, onSave, onCancel }) {
           onChange={handleInputChange}
         />
       </FormGroup>
+      <FormGroup>
+        <Label htmlFor="image">Bild:</Label>
+        <Input
+          type="file"
+          id="image"
+          name="image"
+          accept="image/*"
+          onChange={handleImageChange}
+        />
+        {editedItem.image && <Image src={editedItem.image} alt="Bild" />}
```

Comment:
I am curious: with this line you first check if `editedItem.image` is available and that is true, `<Image />` is displayed?
Is there any difference in writing it like this: `editedItem.image ? <Image /> : null`
But anyways, this is a great approach to handle this!

I just wonder if you need to set this condition, since if a source of an image can't be displayed, its `alt` property is displayed, isn't it? But I guess if you want to not display anything at all in this case, it is perfectly fine.

Reply:
> I am curious: with this line you first check if `editedItem.image` is available and if that is true, `<Image />` is displayed? Is there any difference in writing it like this: `editedItem.image ? <Image /> : null`? But anyways, this is a great approach to handle this!
> 
> I just wonder if you need to set this condition, since if a source of an image can't be displayed, its `alt` property is displayed, isn't it? But I guess if you want to not display anything at all in this case, it is perfectly fine.

Thank you for the review David 🙂. You are absolutely right. Of course, that makes more sense. I will change it.